plugins/acpi-facp/fu-plugin-acpi-facp.c

```diff
@@ -43,6 +43,7 @@ fu_plugin_acpi_facp_add_security_attrs(FuPlugin *plugin, FuSecurityAttrs *attrs)
 	}
 	if (!fu_acpi_facp_get_s2i(facp)) {
 		fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_NOT_ENABLED);
+		fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_ACTION_CONFIG_FW);
```

Comment:
Not always, not all systems offer it...

Reply:
Ya, this would have some kind of "you may be able to" wording.

Reply:
I think that you may be able to wording will need to apply to all 3 resolutions then.

Reply:
on the logic that you might need a new OS kernel?

Reply:
Think about how fwupd can be part of an embedded image, traditional distro, OS Tree, etc.  It will be a different solution depending on how it's packaged.  So the logic here is that you need to fix the software configuration, but you might not be able to actually accomplish this yourself and might have to contact whatever entity provides your software .